core/src/components/header/header-dropdown-list-user/header-dropdown-list-user.tsx

```diff
@@ -1,5 +1,10 @@
 import { Component, h, Prop, Element, Host } from '@stencil/core';
 
+/**
+ * @slot thumbnail - Slot for the thumbnail.
+ * @slot header - Slot for the header.
+ * @slot thumbnail - Slot for the thumbnail.
```

Comment:
```suggestion
```

I think the extra thumbnail was a mistake, right?

Reply:
Correctomundo!